### PR TITLE
Show page

### PR DESCRIPTION
**Pull request by sethcrist**

### CODE REVIEW

**Comment by sethcrist** [comment]
@sjproctor

File: src/App.css [hunk 1]
```diff
@@ -78,4 +78,97 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (sjproctor) [inline src/App.css]: what's going on with all these spaces?

File: src/components/HorrorMovies.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import useHorrorMovies from "../tmdbService";
 
 const HorrorMovies = () => {
   const { movies, isLoading, error } = useHorrorMovies()
+  console.log(movies);
```
Comment (sjproctor) [inline src/components/HorrorMovies.js]: make sure you remove all logs before pushing to main

File: src/pages/Show.js [hunk 3]
```diff
@@ -1,11 +1,121 @@
...
+  return
+<div>
+  Loading...
+</div>
+
+if(!movie)
+return
+<div>
+  No movie found.
+</div>
```
Comment (sjproctor) [inline src/pages/Show.js]: like 47 through 55 needs to be indented

**Review by sjproctor — COMMENTED** [review]
couple small formatting issues

**Review by sjproctor — APPROVED** [review]

**Pull request merged**